### ISSUE TITLE
Create data ingestion pipeline

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#32]
## User story
1. As a {Backend Developer}, I want to set up a data ingestion pipeline that can handle CSV files, so that we can populate our database with the initial geospatial data.
2. I want {a system that automatically processes incoming CSV files, validates the data, and transforms it as necessary.}
3. So that {the data is reliably ingested into our system with high data quality and integrity, ready for queries and analysis.}

## Acceptance criteria
* A scalable data ingestion system is set up to handle .csv files from multiple sources.
* The system automatically processes, validates, and transforms incoming .csv data to fit the database schema.
* Error handling is robust, processing corrupt or incorrect data formats without manual intervention.
* Data ingestion processes are logged for monitoring and debugging purposes.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
